rake_nltk/rake.py

```diff
@@ -31,6 +31,8 @@ def __init__(
         punctuations=None,
         language="english",
         ranking_metric=Metric.DEGREE_TO_FREQUENCY_RATIO,
+        max_length=5,
+        min_length=1,
     ):
         """Constructor.
```

Comment:
Please extend the constructor params with the newly added params. Also add the one that I have left out when you are at it 😄

Reply:
I'm not so good at english, how do you think about these?

```
max_length: Maximum limit of the number of words in a phrase
min_length: Minimum limit of the number of words in a phrase
```

Reply:
@LetianFeng : Your effort is much appreciated. You can leave the documentation and the above. I will take care of the same. 🚀